extensions/gopass/src/index.tsx

```diff
@@ -1,8 +1,9 @@
 import { Action, ActionPanel, Clipboard, closeMainWindow, Icon, List, showHUD, showToast, Toast } from "@raycast/api";
 import { useEffect, useState } from "react";
 import gopass from "./gopass";
+import Details from "./details";
 
-async function copyPassword(entry: string): Promise<void> {
+export async function copyPassword(entry: string): Promise<void> {
```

Comment:
Since you already created a `utils.ts`, I think it would be a good fit for `copyPassword` and `pastePassword`. That would also clean up `index.ts` a little bit. What do you think?